### DISCUSSION TITLE
rabbit mq ，Unable to use after successful configuration

### DISCUSSION THREAD

**Discussion by zhanglidonga** [apache/cloudstack#9048] [Q&A] [answered]
﻿
##### ISSUE TYPE

* Bug Report
﻿
﻿
##### COMPONENT NAME

~~~
rabbit mq
~~~
﻿
##### CLOUDSTACK VERSION

﻿
~~~
4.18
~~~
﻿
##### CONFIGURATION

﻿
﻿
##### OS / ENVIRONMENT

﻿
﻿
##### SUMMARY

﻿
﻿
##### STEPS TO REPRODUCE

﻿

~~~
﻿
~~~
﻿

﻿
##### EXPECTED RESULTS

﻿
~~~
﻿You can receive notification messages from the system, such as the stop or start of a virtual machine, and the success or failure of creating a virtual machine
~~~
﻿
##### ACTUAL RESULTS

﻿

~~~
﻿All operations did not send any messages
~~~
﻿

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by DaanHoogland** [comment]
@zhanglidonga , are you sure you have the configuration right? Can you add the steps to reproduce section to the ticket?

**Comment by zhanglidonga** [comment]
![WechatIMG13](https://github.com/apache/cloudstack/assets/30895730/cc1197d1-08ff-4417-bcac-ab217c426e72)
I can see the link information on RabbitMQ, but there is no traffic
![WechatIMG14](https://github.com/apache/cloudstack/assets/30895730/2d8a3089-9899-4862-a799-171339d6ccce)

**Comment by zhanglidonga** [comment, marked as answer]
This part of the platform's settings has already been opened. Are there any other missing settings?
![WechatIMG15](https://github.com/apache/cloudstack/assets/30895730/3f6c0f6b-9d34-4506-bff2-a5a8656a9a64)

**Comment by zhanglidonga** [comment]
Thank you, I have resolved this issue
